### DISCUSSION TITLE
Macos 安装 Permission denied

### DISCUSSION THREAD

**Discussion by Robin-sys** [mamoe/mirai#1280] [Q&A] [answered]
在terminal里运行这个文件的时候就报错了...

Robins-MacBook-Pro:qqbot apple$ ./mcl-installer-72b890d-macos-amd64
-bash: ./mcl-installer-72b890d-macos-amd64: Permission denied

我搜了一下也没有找到直接对应的解决方法，想问一下大大这个怎么解决？
能回复的话万分感谢！

**Comment by Him188** [comment, marked as answer]
`chmod +x *`

**Reply by Robin-sys** [reply]
非常感谢！
